useful-scripts/misc-test-utils/src/index.ts

```diff
@@ -14,6 +14,7 @@ import createPublication from './modules/publication/createPublication';
 import addReleaseTextContentBlock from './modules/release/addContentBlock';
 import themeService from './services/themeService';
 import topicService from './services/topicService';
+import Guid from './utils/Guid';
```

Comment:
Does this import actually work? The files still seem to be lowercase `guid.ts` rather than `Guid.ts`

Reply:
hmm, that's strange (not seeing that). I've just ran `git rm -r --cached misc-test-utils` to try to get git to pick up the changes in file casing. Not sure if this has sorted it?